New version: Reactant_jll v0.0.16+0

jlbuild opened the pull request:
Autogenerated JLL package registration

* Registering JLL package Reactant_jll.jl
* Repository: https://github.com/JuliaBinaryWrappers/Reactant_jll.jl
* Version: v0.0.16+0
* Commit: 15a741ac29499f5bd4b0abf102c7621bc079a71d
* Revision on Yggdrasil: https://github.com/JuliaPackaging/Yggdrasil/commit/c49ca33711e1a54a2340e49e13a0153eb4a24513
* Created by: @wsmoses

wsmoses commented:
@giordano @vchuravy @gbaraldi uh, what's happening here?

```julia
┌ Reactant_jll
│  WARNING: redefinition of constant Reactant_jll.preferences. This may fail, cause incorrect answers, or produce other errors.
│  WARNING: Method definition augment_platform!(Base.BinaryPlatforms.Platform) in module Reactant_jll at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/.pkg/platform_augmentation.jl:69 overwritten at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/.pkg/platform_augmentation.jl:84.
│  ERROR: Method overwriting is not permitted during Module precompilation. Use `__precompile__(false)` to opt-out of precompilation.
└  
WARNING: redefinition of constant Reactant_jll.preferences. This may fail, cause incorrect answers, or produce other errors.
WARNING: Method definition augment_platform!(Base.BinaryPlatforms.Platform) in module Reactant_jll at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/.pkg/platform_augmentation.jl:69 overwritten at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/.pkg/platform_augmentation.jl:84.
ERROR: Method overwriting is not permitted during Module precompilation. Use `__precompile__(false)` to opt-out of precompilation.
WARNING: redefinition of constant Reactant_jll.preferences. This may fail, cause incorrect answers, or produce other errors.
ERROR: LoadError: UndefVarError: `builtins` not defined
Stacktrace:
  [1] top-level scope
    @ /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/src/wrappers/x86_64-linux-gnu-cxx11-mode+opt.jl:2
  [2] include(mod::Module, _path::String)
    @ Base ./Base.jl:495
  [3] top-level scope
    @ /tmp/jl_yk821u/packages/JLLWrappers/inPJn/src/toplevel_generators.jl:192
  [4] include(mod::Module, _path::String)
    @ Base ./Base.jl:495
  [5] _require(pkg::Base.PkgId, env::String)
    @ Base ./loading.jl:2014
  [6] __require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:1812
  [7] #invoke_in_world#3
    @ ./essentials.jl:926 [inlined]
  [8] invoke_in_world
    @ ./essentials.jl:923 [inlined]
  [9] _require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:1803
 [10] macro expansion
    @ ./loading.jl:1790 [inlined]
 [11] macro expansion
    @ ./lock.jl:267 [inlined]
 [12] __require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1753
 [13] #invoke_in_world#3
    @ ./essentials.jl:926 [inlined]
 [14] invoke_in_world
    @ ./essentials.jl:923 [inlined]
 [15] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:1746
in expression starting at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/src/wrappers/x86_64-linux-gnu-cxx11-mode+opt.jl:2
in expression starting at /tmp/jl_yk821u/packages/Reactant_jll/L2CKO/src/Reactant_jll.jl:2
┌ Error: Was not able to successfully `import` the package
└ @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/guidelines.jl:972
┌ Info: Version can be `import`ed
│   meets_this_guideline = false
└   message = "I was not able to load the package (i.e. `import Reactant_jll` failed). See the AutoMerge logs for details."
ERROR: LoadError: RegistryCI.AutoMerge.AutoMergeGuidelinesNotMet("The automerge guidelines were not met.")
Stacktrace:
 [1] pull_request_build(data::RegistryCI.AutoMerge.GitHubAutoMergeData; check_license::Bool, new_package_waiting_period::Day)
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/pull_requests.jl:257
 [2] pull_request_build
   @ ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/pull_requests.jl:174 [inlined]
 [3] pull_request_build(api::GitHub.GitHubWebAPI, pr_number::Int64, current_pr_head_commit_sha::String, registry::GitHub.Repo, registry_head::String; whoami::String, auth::GitHub.OAuth2, authorized_authors::Vector{String}, authorized_authors_special_jll_exceptions::Vector{String}, error_exit_if_automerge_not_applicable::Bool, master_branch::String, master_branch_is_default_branch::Bool, suggest_onepointzero::Bool, point_to_slack::Bool, registry_deps::Vector{String}, check_license::Bool, public_registries::Vector{String}, read_only::Bool, environment_variables_to_pass::Vector{String}, new_package_waiting_period::Day)
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/pull_requests.jl:169
 [4] pull_request_build
   @ ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/pull_requests.jl:72 [inlined]
 [5] run(; env::Base.EnvDict, cicfg::RegistryCI.AutoMerge.GitHubActions, merge_new_packages::Bool, merge_new_versions::Bool, new_package_waiting_period::Day, new_jll_package_waiting_period::Minute, new_version_waiting_period::Minute, new_jll_version_waiting_period::Minute, registry::String, tagbot_enabled::Bool, authorized_authors::Vector{String}, authorized_authors_special_jll_exceptions::Vector{String}, additional_statuses::Vector{String}, additional_check_runs::Vector{String}, error_exit_if_automerge_not_applicable::Bool, master_branch::String, master_branch_is_default_branch::Bool, suggest_onepointzero::Bool, point_to_slack::Bool, registry_deps::Vector{String}, api_url::String, check_license::Bool, public_registries::Vector{String}, read_only::Bool, environment_variables_to_pass::Vector{String})
   @ RegistryCI.AutoMerge ~/.julia/packages/RegistryCI/mNCS2/src/AutoMerge/public.jl:144
 [6] top-level scope
   @ ~/work/_temp/bc075bc3-678f-498e-8699-2c1df87a7a6c:4
in expression starting at /home/runner/work/_temp/bc075bc3-678f-498e-8699-2c1df87a7a6c:4
```

giordano commented:
There's a duplicate `preferences`: https://github.com/JuliaBinaryWrappers/Reactant_jll.jl/blob/6017e7223838f8d63e5f31355b1a0fa7659c0076/.pkg/platform_augmentation.jl#L4 and https://github.com/JuliaBinaryWrappers/Reactant_jll.jl/blob/6017e7223838f8d63e5f31355b1a0fa7659c0076/.pkg/platform_augmentation.jl#L29.  But I presume you figured it out already: https://github.com/JuliaPackaging/Yggdrasil/pull/9338

wsmoses commented:
Eh I didn’t actually figure that out but thought removing that code would
be wise and maybe fix

On Sat, Aug 31, 2024 at 1:19 PM Mosè Giordano ***@***.***>
wrote:

> There's a duplicate preferences:
> https://github.com/JuliaBinaryWrappers/Reactant_jll.jl/blob/6017e7223838f8d63e5f31355b1a0fa7659c0076/.pkg/platform_augmentation.jl#L4
> and
> https://github.com/JuliaBinaryWrappers/Reactant_jll.jl/blob/6017e7223838f8d63e5f31355b1a0fa7659c0076/.pkg/platform_augmentation.jl#L29.
> But I presume you figured it out already: JuliaPackaging/Yggdrasil#9338
> <https://github.com/JuliaPackaging/Yggdrasil/pull/9338>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaRegistries/General/pull/114251#issuecomment-2323004329>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJTUXCCECD462XWNM7NWL3ZUICLXAVCNFSM6AAAAABNN573GOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGMRTGAYDIMZSHE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

jlbuild committed:
New version: Reactant_jll v0.0.16+0

UUID: 0192cb87-2b54-54ad-80e0-3be72ad8a3c0
Repo: https://github.com/JuliaBinaryWrappers/Reactant_jll.jl.git
Tree: 31120ca0c568d1792b5c2a55aafe320cfe900969

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
